### PR TITLE
QE: Printing node OS in twopence_init.rb

### PR DESCRIPTION
**Pull request by elariekerboull**
## What does this PR change?

Prints the OS version of each node in the testsuite at runtime.

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Links

Fixes # https://github.com/SUSE/spacewalk/issues/18722
Tracks # 
4.2
4.3

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by srbarrios — APPROVED** [review]

File: testsuite/features/support/twopence_init.rb [hunk 1]
```diff
@@ -131,6 +131,8 @@ def twopence_init(target)
   node.init_full_hostname(fqdn)
 
   STDOUT.puts "Host '#{$named_nodes[node.hash]}' is alive with determined hostname #{hostname.strip} and FQDN #{fqdn.strip}" unless $build_validation
+  result, _code = node.run('grep PRETTY /etc/os-release')
+  puts "'#{$named_nodes[node.hash]}' is running OS #{result}"
```
Comment (srbarrios) [inline testsuite/features/support/twopence_init.rb:135]: Fine for now.
I think we can have in the future an improvement, and have a new attribute in Lavanda class, that contains interesting info from that file.

And then do something like:
node.os_release(result)

INitializing for instance attributes like:
node.os_release_id
node.os_release_version_id


Getting rid of this common called method:
```
def get_os_version(node)
  os_family_raw, code = node.run('grep "^ID=" /etc/os-release', check_errors: false)
  return nil, nil unless code.zero?
  os_family = os_family_raw.strip.split('=')[1]
  return nil, nil if os_family.nil?
  os_family.delete! '"'
  os_version_raw, code = node.run('grep "^VERSION_ID=" /etc/os-release', check_errors: false)
  return nil, nil unless code.zero?
  os_version = os_version_raw.strip.split('=')[1]
  return nil, nil if os_version.nil?
  os_version.delete! '"'
  # on SLES, we need to replace the dot with '-SP'
  os_version.gsub!(/\./, '-SP') if os_family =~ /^sles/
  STDOUT.puts "Node: #{node.hostname}, OS Version: #{os_version}, Family: #{os_family}"
  [os_version, os_family]
end
```

Comment (Bischoff) [inline testsuite/features/support/twopence_init.rb:135]: You could also have used `node.full_hostname` or `hostname.strip`, but that's probably ok that way too

In the past, `STDOUT.puts` was needed to work around jenkin's buffering, but maybe it's not needed anymore

**Review by Bischoff — APPROVED** [review]
just left remarks

File: testsuite/features/support/twopence_init.rb (hunk 1 above)
Comment (elariekerboull) [reply testsuite/features/support/twopence_init.rb]: Switched `puts` with `STDOUT.puts` just in case and to be consistent with its use two lines above.

Comment (elariekerboull) [reply testsuite/features/support/twopence_init.rb]: I'll create a card with this suggestion

**Pull request merged**